samples/msteams-application-qbot/Build/azure-pipelines-ci-dev.yml

```diff
@@ -0,0 +1,50 @@
+trigger:
```

Comment:
We probably want to drop this file / not publish it. 

It isn't very instructive and uses some tasks from internal tooling

Reply:
Removed.